services/cd-service/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.14
+FROM alpine:3.15
```

Comment:
please also change docker/build/Dockerfile

Reply:
I'm kinda trying to create a new PR because I didn’t mange to get ride of the 
Codereviewr Check error

Reply:
that's fine.
Then I would suggest this: close this PR and add a comment here that links to the new PR